src/AddIns/Uno.UI.Lottie.Skia/ts/lottie-web.d.ts

```diff
@@ -0,0 +1,131 @@
+﻿declare namespace Lottie {
+	export interface AnimationItem {
+		play(): void;
+
+		stop(): void;
+
+		pause(): void;
+
+		// one param speed (1 is normal speed)   
+		setSpeed(speed: number): void;
+
+		// one param direction (1 is normal direction)  
+		setDirection(direction: number): void;
+
+		// If false, it will respect the original AE fps. If true, it will update as much as possible. (true by default)
+		setSubframe(flag: boolean): void;
+
+		// first param is a numeric value. second param is a boolean that defines time or frames for first param        
+		goToAndPlay(value: number, isFrame: boolean): void;
+
+		// first param is a numeric value. second param is a boolean that defines time or frames for first param
+		goToAndStop(value: number, isFrame: boolean): void;
+
+		// first param is a single array or multiple arrays of two values each(fromFrame,toFrame), second param is a boolean for forcing the new segment right away
```

Comment:
Codacy found an issue: [Exceeds maximum line length of 120](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)